src/main/java/net/dries007/tfc/client/ClientEventHandler.java

```diff
@@ -177,6 +179,10 @@ public static void clientSetup(FMLClientSetupEvent event)
                         }
                         return 1.0F;
                     });
+
+                    TFCBlocks.WOODS.values().stream().map(map -> map.get(BARREL).get().asItem()).forEach(barrel -> {
+                        ItemProperties.register(barrel, Helpers.identifier("sealed"), (stack, level, entity, unused) -> stack.hasTag() ? 1.0f : 0f);
+                    });
```

Comment:
Nitpick, `.stream().map().forEach()` is a bit redundant, and could be `.forEach(map -> ... `, using `map.get(BARREL).get().asItem()` directly in the register call.